src/plugins/data_source_management/public/components/create_data_source_wizard/create_data_source_wizard.tsx

```diff
@@ -45,7 +45,7 @@ export const CreateDataSourceWizard: React.FunctionComponent<CreateDataSourceWiz
 
   /* Set breadcrumb */
   useEffectOnce(() => {
-    setBreadcrumbs(getCreateBreadcrumbs());
+    setBreadcrumbs(getCreateOpenSearchDataSourceBreadcrumbs());
```

Comment:
may need to also change the 'cancel behavior; to align with the s3 and prometheus to fall back to the card view panel.

Reply:
Agree. I have applied both the change to both components and the tests